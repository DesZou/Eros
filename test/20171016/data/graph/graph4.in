5 10 10
1 2 359
1 3 342
1 4 825
1 5 777
2 3 712
2 4 13
2 5 901
3 4 23
3 5 111
4 5 48
10
4 2
4 2
4 5
5 1
1 3
2 4
3 4
2 5
5 1
3 5
