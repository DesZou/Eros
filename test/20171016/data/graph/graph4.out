273
273
238
577
602
273
263
165
577
175
